docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
   worker:
     extends:
       service: service_base
-    command: celery worker -E -A inspirehep.celery --loglevel=INFO --purge
+    command: celery worker -E -A inspirehep.celery --loglevel=INFO --purge -Q cerely,migrator
```

Comment:
`cerely` looks wrong :)

Reply:
Yeap, @david-caro fixed in a later PR #3083.